website/docs/r/satellite_cluster.html.markdown

```diff
@@ -16,19 +16,29 @@ Create, update, or delete [IBM Cloud Satellite Cluster](https://cloud.ibm.com/do
 ###  Create satellite cluster
 
 ```hcl
+data "ibm_resource_group" "rg" {
+    name = var.resource_group
```

Comment:
don;t use var.XXX remove them and provide some constant values

Format the example